engine/framework/worker.go

```diff
@@ -236,8 +236,9 @@ func NewBaseWorker(
 		errCenter:            errctx.NewErrCenter(),
 		clock:                clock.New(),
 		businessMetaKVClient: cli,
-		metricFactory:        promutil.NewFactory4Worker(ctx.ProjectInfo, MustConvertWorkerType2JobType(tp), masterID, workerID),
-		logger:               frameLog.WithWorkerID(frameLog.WithMasterID(logger, masterID), workerID),
+		metricFactory: promutil.NewFactory4Worker(ctx.ProjectInfo,
```

Comment:
Is it better to change the function signature to 
```go
func NewFactory4Worker(info tenant.ProjectInfo, prefix string, jobID engineModel.JobID, workerID frameModel.WorkerID,
```
Then we don't need to add the job type `JobTypeNameNone`